configure.ac

```diff
@@ -76,14 +76,14 @@ AC_ARG_WITH([network],
     [Specify network. Default is testnet.]
```

Comment:
I think this should be switched to "Default is main/mainnet"

Reply:
Agreed. I think we should use "main." It matches the expected command line argument.

Reply:
thanks, fixed.